openedx/core/djangoapps/plugin_api/views.py

```diff
@@ -68,9 +66,9 @@ def css_dependencies(self):
         """
         return []
 
-    def add_resource_urls(self, fragment):
+    def add_fragment_resource_urls(self, fragment):
         """
-        Adds URLs for JS and CSS resources that this XBlock depends on to `fragment`.
+        Adds URLs for JS and CSS resources need by this fragment.
```

Comment:
Same "need => needed" comment, but adding it to a review if that helps manage the comments.